Extension/src/LanguageServer/extension.ts

```diff
@@ -150,7 +150,12 @@ export async function getBuildTasks(returnComplerPath: boolean): Promise<vscode.
     // for the active file, remove duplicate compiler names, then finally add the user's compilerPath setting.
     let compilerPaths: string[];
     const isWindows: boolean = os.platform() === 'win32';
-    const activeClient: Client = getActiveClient();
+    let activeClient: Client;
+    try {
+        activeClient = getActiveClient();
+    } catch (e) {
+        return []; // Language service features may be disabled.
```

Comment:
Do you want to log it to the output when you catch it as a warning in case it isn't the exception you are expecting?

Reply:
It's not something I'm currently concerned about, but it was easy to add a warning.